package.json

```diff
@@ -28,6 +28,7 @@
     "@salesforce/kit": "^1.5.32",
     "@salesforce/ts-types": "^1.4.2",
     "archiver": "^5.3.0",
```

Comment:
This is the third party dependency that got the deal done. https://www.npmjs.com/package/deasync. MIT licensed and around 1M downloads a week so seemed a safe choice.  I'm following up on what I need to do for 3PP land.